perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/client/ExceptionResolverImpl.java

```diff
@@ -190,6 +194,19 @@ private void resolve() {
 	}
 
 
+	private void resolveNoPrefilledUneditableRequiredDataException() {
+
+		String emptyItems = "<ul>";
+		if (!exception.getFormItems().isEmpty()) {
+			for (ApplicationFormItemData item : exception.getFormItems()) {
+				emptyItems += "<li>" + trans.requiredEmptyItem(item.getFormItem().getShortname()) + "</li>";
```

Comment:
I'm wondering what is more understandable for users, since there might be more ways how the item is recognized. Shortname of form item is arbitrary name chosen by the VO/Group manager. We have then name displayed on the form and then destinations/source attribute.

For now shortname will do I guess. I just wonder, if displaying `Required empty item:` from the translation on each line is necessary a list of items is explained by the sentence above it.

Reply:
We can discuss on review, which option is the best for users (short name, display name, etc). 
Texts `Required empty items:` and `Missing attribute:` removed from these lists.